.github/workflows/test.yml

```diff
@@ -83,6 +83,9 @@ jobs:
               database: psycopg3
             - python-version: '3.12'
               database: psycopg3
+            - python-version: '3.13'
+              # psycopg installs version 3+; psycopg3 not found via pip in python 3.13
+              database: psycopg
```

Comment:
Can you explain why you're using `psycopg` instead of `psycopg3` here and below? This doesn't look correct to me. psycopg3 doesn't refer to the PyPI package, it's just an internal name we're using to differentiate between the old psycopg2 and the new psycopg v3; the package name itself is defined here: https://github.com/jazzband/django-debug-toolbar/blob/244d4fd7f94caf77c4c18058420fa03f21751195/tox.ini#L16

Reply:
Yeah, I got that wrong, sorry.